tests/ui/test_ui_tools.py

```diff
@@ -741,65 +711,35 @@ def test_keypress_GO_RIGHT(self, mocker, topic_view, key, widget_size):
     def test_keypress_SEARCH_TOPICS(self, mocker, topic_view, key, widget_size):
         size = widget_size(topic_view)
         mocker.patch(VIEWS + ".TopicsView.set_focus")
+        topic_view.log.extend(["FOO", "foo", "fan", "boo", "BOO"])
+        topic_view.log.set_focus(3)
+
         topic_view.keypress(size, key)
         topic_view.set_focus.assert_called_once_with("header")
         topic_view.header_list.set_focus.assert_called_once_with(2)
+        assert topic_view.focus_index_before_search == 3
 
     @pytest.mark.parametrize("key", keys_for_command("GO_BACK"))
     def test_keypress_GO_BACK(self, mocker, topic_view, key, widget_size):
         size = widget_size(topic_view)
         mocker.patch(VIEWS + ".TopicsView.set_focus")
         mocker.patch.object(topic_view.topic_search_box, "reset_search_text")
-        topic_view.keypress(size, key)
-        topic_view.set_focus.assert_called_once_with("body")
-        assert topic_view.topic_search_box.reset_search_text.called
-        assert topic_view.log == self.topics_btn_list
-
-    @pytest.mark.parametrize("search_topics_key", keys_for_command("SEARCH_TOPICS"))
-    @pytest.mark.parametrize("go_back_key", keys_for_command("GO_BACK"))
-    @pytest.mark.parametrize(
-        "current_focus, topic",
-        [
-            (0, "FOO"),
-            (2, "fan"),
-            (4, "BOO"),
-        ],
-    )
-    def test_return_to_focus_after_search(
```

Comment:
So the problem is that it only enters the search bar and not the search results, I don't think there is a way to use the 'down' keypress to change focus inside the log, I was getting errors. Hence I assumed I was already inside the search results and was escaping from there which is also shorter and can be done inside GO_BACK

Reply:
I just realized why I couldn't get 'down' to work.
But since all of the code happens inside SEARCH_STREAMS and GO_BACK unlike before. It makes sense to write these tests inside GO_BACK as this is what is responsible for resetting state and also it's much shorter and cleaner and tests the same functionality